docs/_pages/config-json.md

```diff
@@ -151,11 +151,11 @@ Example of <i>**scheduler**</i>
 
 ### Communication options
 
-<i>**fp32_allreduce**</i>: [boolean]
+<i>**communication_data_type**</i>: [boolean]
 
-| Description                                                    | Default |
-| -------------------------------------------------------------- | ------- |
-| During gradient averaging perform allreduce with 32 bit values | `false` |
+| Description                                                             | Default |
+| ----------------------------------------------------------------------- | ------- |
+| During gradient averaging perform communication with selected data type |  `none` |
```

Comment:
I think it is better to make the default communication data type fp16 and completely drop the none option. What do you think?